cs/routing.texy

```diff
@@ -143,24 +143,40 @@ $route = new Route('//www.%sld%.%tld/%basePath%/<presenter>/<action', ...);
 
 
 
-Kolekce rout
-------------
-
-Protože rout budeme zpravidla definovat více než jednu, zabalíme je do [RouteList |api:Nette\Application\Routers\RouteList]. A opět rovnou ukázka použítí v souboru [bootstrap.php |presenters#index-php-bootstrap-php]: (`$container` je [systémový DI kontejner |configuring#sluzby])
+Router Factory
```

Comment:
Mozna jenom detail, ale co tam nedavat tu mezeru, at to koresponduje primo s nazvem tridy?